tracks/go/exercises/scrabble-score/README.md

```diff
@@ -0,0 +1,90 @@
+# Scrabble Score
+
+This is the first more complex task. There are several ways to solve this and benchmarks are 
+one important criteria for a good solution. Another important criteria is simple and readable code.
+
+## Reasonable solutions
+
+Solutions have a wide range here. A solution should be in the range of the solutions below to be approved. That means not 
+overly complicated / verbose and not slower than the second solution. Allocations are not necessary and should be avoided.
+
+_Unformatted or unlinted code is a reason not to approve._
+
+Fast solution:
+```
+// Score impelents scrabble score
+func Score(s string) int {
+	var score int
+	for _, r := range s {
+		switch unicode.ToUpper(r) {
+		case 'A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T':
+			score += 1
+		case 'D', 'G':
+			score += 2
+		case 'B', 'C', 'M', 'P':
+			score += 3
+		case 'F', 'H', 'V', 'W', 'Y':
+			score += 4
+		case 'K':
+			score += 5
+		case 'J', 'X':
+			score += 8
+		case 'Q', 'Z':
+			score += 10
+		}
+	}
+
+	return score
+}
+```
+
+Very readable solution but ~3 times slower:
+```
+var m = map[rune]int{
+	'A': 1, 'E': 1, 'I': 1, 'O': 1, 'U': 1, 'L': 1, 'N': 1, 'R': 1, 'S': 1, 'T': 1,
+	'D': 2, 'G': 2,
+	'B': 3, 'C': 3, 'M': 3, 'P': 3,
+	'F': 4, 'H': 4, 'V': 4, 'W': 4, 'Y': 4,
+	'K': 5,
+	'J': 8, 'X': 8,
+	'Q': 10, 'Z': 10,
+}
+
+// Score impelents scrabble score
+func Score(s string) int {
+	var score int
+	for _, r := range s {
+		score += m[unicode.ToUpper(r)]
+	}
+
+	return score
+}
+```
+
+## Common suggestions
+
+The most common feedback revolves around:
+
+* Simplicity and readability
+* Speed and allocations
+
+**Simplicity and Readability**
+
+* are they using a lot of if statements instead of a switch? Suggest to use a switch for readability.
+
+Sometimes we have to make compromises between speed on the one side and simplicity / readability on the other. In this exercise this is not the case. My suggestion is to make this exercise mostly about speed and the solution will become simple and readable.
```

Comment:
But then the description needs to be changed:

> If the test suite contains benchmarks, you can run these ...

This implies that you _can_ use the benchmarks.